parker_harding.yml

```
@@ -0,0 +1,10 @@
+name: "Parker Harding"
+image: "images/parker_harding.jpg"
+title: "Associate IT Professional"
+website: "https://www.linkedin.com/feed/"
+institution: "Job Rotation Program/Center for High Throughput Computing"
```

```suggestion
institution: University of Wisconsin–Madison"
```

Here we mean the UW (some CHTC staff work for the Morgridge Institute)